docker-compose.yaml

```diff
@@ -1,6 +1,6 @@
 version: '3.9'
 
-services:
+uservices:
```

Comment:
Why have u changed services to uservices

Reply:
+dlfhjignp;edwo9n0i gm,as,d'n gnm,adsf bmn;lgl'

-+_=-Poelw vo;